src/sidebar/components/FilterStatus.js

```diff
@@ -49,271 +26,234 @@ import { useRootThread } from './hooks/use-root-thread';
  */
 
 /**
- * @typedef FilterModeProps
- * @prop {FilterState} filterState
- * @prop {Thread} rootThread
- */
-
-/**
- * Render information about the currently-applied filters and a button that
- * allows the user to clear filters (or toggle user-focus mode on and off).
+ * Render status text describing the currently-applied filters.
  *
- * @param {FilterStatusPanelProps} props
+ * @param {FilterStatusMessageProps} props
  */
-function FilterStatusPanel({
-  actionButton,
+function FilterStatusMessage({
```

Comment:
I noticed that the query phrase appears in single quotes (eg. "No results for 'foo'"). Should it use double quotes?